modules/tinymce/src/core/test/ts/browser/dom/ControlSelectionTest.ts

```diff
@@ -56,7 +56,7 @@ describe('browser.tinymce.core.dom.ControlSelectionTest', () => {
     const dimension = Css.getRaw(element, name).orThunk(() => Attribute.getOpt(element, name))
       .map((v) => parseInt(v, 10))
       .getOr(0);
-    assert.isBelow(Math.abs(dimension - expectedDimension), 3, label + ` ${dimension}px ~= ${expectedDimension}px`);
+    assert.approximately(dimension, expectedDimension, 3, `${label} ${dimension}px ~= ${expectedDimension}px`);
```

Comment:
Does `assert.approximately` include the `a ~= b` part in the message automatically? Maybe we could take this down to just `label`

Reply:
https://github.com/chaijs/chai/blob/main/lib/chai/core/assertions.js#L2979 

It looks like it does

Reply:
We've used this style everywhere else, as while it is included it's more readable in this format. If we want to change it, that's out of scope and something we should do via a separate PR.